Leaderboard/views/index.hbs

```diff
@@ -1,2 +1 @@
-<h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+
```

Comment:
We probably want to have some form of welcome page here explaining the application.